bin/tempo/src/tempo_cmd.rs

```diff
@@ -0,0 +1,97 @@
+use std::path::PathBuf;
+
+use clap::{Parser, Subcommand, error::ErrorKind};
+use commonware_cryptography::{PrivateKeyExt as _, Signer, ed25519::PrivateKey};
+use eyre::Context;
+use tempo_commonware_node_config::SigningKey;
+
+#[derive(Debug, Parser)]
+#[command(name = "tempo")]
+struct TempoCli {
+    #[command(subcommand)]
+    command: TempoCommand,
+}
+
+#[derive(Debug, Subcommand)]
+enum TempoCommand {
+    /// Consensus-related commands.
+    Consensus(ConsensusCommand),
+}
+
+#[derive(Debug, clap::Args)]
+struct ConsensusCommand {
+    #[command(subcommand)]
+    command: ConsensusSubcommand,
+}
+
+#[derive(Debug, Subcommand)]
+enum ConsensusSubcommand {
+    /// Generates an ed25519 signing key pair to be used in consensus.
+    GenerateSigningKey(GenerateSigningKey),
+    /// Calculates the public key from an ed25519 signing key.
+    CalculatePublicKey(CalculatePublicKey),
```

Comment:
```suggestion
    CalculateVerificationKey(CalculateVerificationKey),
```

Should use `signing <> verification` or `private <> public`. Mixing is weird

Reply:
ok, will change to private <> public, since thats what most operators will be familiar with. good call

Reply:
Only caveat of course being that are using `--consensus.signing-key`. So the symmetry here makes sense. I let you do decide. :-)